sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidCertainMethodCheck.java

```diff
@@ -173,6 +173,20 @@ public void visitToken(DetailAST ast) {
         }
     }
 
+    /**
+     * Count the parameters given to a method call.
+     * @param expressionList The expression list containing the parameters.
+     * @return The number of parameters.
+     */
+    private int getMethodCallParameterCount(DetailAST expressionList) {
```

Comment:
getMethodCallParameterCount vs expressionList ... mismatch

please make this method to be based on method AST, like `getMethodCallParameterCount(ast)` , and please make  method as static, one dat it will be moved to some Util class.